lib/foreman_ansible/engine.rb

```diff
@@ -10,6 +13,13 @@ class Engine < ::Rails::Engine
     config.autoload_paths += Dir["#{config.root}/app/services"]
     config.autoload_paths += Dir["#{config.root}/app/views"]
 
+    initializer 'foreman_docker.register_gettext', :after => :load_config_initializers do
```

Comment:
Yeah, it doesn't really matter unless you have both foremandocker & foremanansible installed